### PR TITLE
Fix typos in documentation

### PR DESCRIPTION
**Pull request by tiffany76**
__Pull Request Description__

This PR fixes minor typographical errors, such as misspelled words, omitted or extra words, and missing punctuation, in all of the Markdown doc files. 

These changes will improve the clarity of the documentation for FarmData2 users.

__Licensing Certification__

FarmData2 is a [Free Cultural Work](https://freedomdefined.org/Definition) and all accepted contributions are licensed as described in the LICENSE.md file. This requires that the contributor holds the rights to do so. By submitting this pull request __I certify that I satisfy the terms of the [Developer Certificate of Origin](https://developercertificate.org/)__ for its contents.

### CODE REVIEW

**Comment by braughtg** [comment]
Thank you for helping to cleanup this documentation!  It's great to have more eyes on it.  

This all looks good so far.   When you are finished and would like this PR to be reviewed, flip it from a draft PR to a full PR, and tag me in a comment.

File: docs/Technologies.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ The FarmData2 code and documentation can be edited with any editor.  However, fo
 
 TODO: Write this for VSCodium.
 
-<!-->
+<!--
```
Comment (tiffany76) [inline docs/Technologies.md:58]: I assumed lines 58-76 were meant to be commented out, so I fixed the opening tag.

Comment (braughtg) [reply docs/Technologies.md:58]: That is correct.  Good catch!

**Comment by tiffany76** [comment]
Hi @braughtg,
This PR is ready for review. I know it's pretty large. The changes are all small, but let me know if you'd like me to break it up into two PRs.
Many thanks.

**Comment by braughtg** [comment]
This is great just as it is!  Thank you @tiffany76 for your amazingly thorough review and cleanup of the documentation.  That is super helpful and most appreciated.  Please feel free to browse the issue tracker or get in touch in Zulip if you are interested in finding other things to do in FarmData2!  We welcome any future contributions you might be interested in making.

**Pull request merged**